src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiK8sVngSpec.java

```diff
@@ -0,0 +1,390 @@
+package com.spotinst.sdkjava.model.api.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiK8sVngSpec implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>                      isSet;
+    private String                           id;
+    private String                           name;
+    private Boolean                          associatePublicIpAddress;
+    private ApiK8sVngAutoScaleSpec 			 autoScale;
+    private List<ApiK8sVngBlockDevice>       blockDeviceMappings;
+    private ApiK8sVngElasticIpPool			 elasticIpPool;
+    private ApiK8sVngIamInstanceProfileSpec  iamInstanceProfile;
+    private String                           imageId;
+    private ApiK8sVngInstanceMetadataOptions instanceMetadataOptions;
+    private List<String>                     instanceTypes;
+    private List<ApiK8sVngLabels>			 labels;
+    private String                           oceanId;
+    private List<String>                     preferredSpotTypes;
+    private ApiK8sVngResourceLimits			 resourceLimits;
+    private Boolean                          restrictScaleDown;
+    private Integer                          rootVolumeSize;
+    private List<String>                  	 securityGroupIds;
+    private ApiK8sVngStrategySpec  			 strategy;
+    private List<ApiK8sVngTaints>            taints;
+    private List<String>         		  	 subnetIds;
+    private List<ApiK8sVngTag>               tags;
+    private String                        	 userData;
+    private Date                          	 createdAt;
```

Comment:
Do we need to have these parameters - createdAt and updatedAt

Reply:
Remove the parameters as they are not show up in postman